samples/bot-conversation/csharp/README.md

```diff
@@ -174,5 +186,4 @@ To learn more about deploying a bot to Azure, see [Deploy your bot to Azure](htt
 
 ## Further reading
 
-- [How Microsoft Teams bots work](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-basics-teams?view=azure-bot-service-4.0&tabs=javascript)
-
+- [How Microsoft Teams bots work](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-basics-teams?view=azure-bot-service-4.0&tabs=javascript)
```

Comment:
Did we added these methods related doc or SDK link here?

Reply:
The official doc is not released yet publicly. We will add it when doc is published